app/workers/delete_audits_worker.rb

```diff
@@ -0,0 +1,8 @@
+class DeleteAuditsWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :low_priority
+
+  def perform(audit_ids)
+    Audited::Audit.delete(audit_ids)
```

Comment:
using `delete` will instantiate every audit, `delete_all` will delete via SQL which will be much quicker